Gems/Atom/RPI/Code/Source/RPI.Builders/Material/MaterialBuilder.cpp

```diff
@@ -113,9 +113,10 @@ namespace AZ
 
             if (!parentMaterialPath.empty())
             {
-                // Register dependency on the parent material source file so we can load it and use it's data to build this variant material.
-                // Note, we don't need a direct dependency on the material type because the parent material will depend on it.
-                MaterialBuilderUtils::AddPossibleDependencies(request.m_sourceFile,
+                // Register dependency on the parent material source file so we can load it and use it's data to build this variant
+                // material. Note, we don't need a direct dependency on the material type because the parent material will depend on it.
+                MaterialBuilderUtils::AddPossibleDependencies(
```

Comment:
This comment seems to conflict with the comment below the if, One says a dependecy on the material type is not needed and the other says it is.